config.example.yaml

```diff
@@ -213,6 +225,9 @@ services:
       salt: secret
       default_page_size: 250
       user_agent: RESTBase
+      internal_request_whitelist:
```

Comment:
@gwicke We don't declare a special spec for a [security requirement scheme](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#securitySchemeObject) now, because swagger doesn't allow custom object types and what they have doesn't fit our needs: they have only 'basic', 'apiKey' and 'oAuth2' types, and what we have is non of these. Also, although internal requests are now used only in security part, it's not just about it, I could easily imagine us using it for some other stuff like counters/logging/whatever else.